ringo_printtemplate/views.py

```diff
@@ -78,9 +79,11 @@ def save_file(request, item):
         request.POST.get('file').file.seek(0)
         data = request.POST.get('file').file.read()
         filename = request.POST.get('file').filename
+        item.mime = check_mime_type_from_buffer(data)
+        if item.mime != "application/vnd.oasis.opendocument.text":
+            raise AttributeError()
```

Comment:
Is there any chance this AttributeError is ever thrown? As this method is only called on a POST request after the validation already happended I can not think of any situation where this check could fail. Do you?